### DISCUSSION TITLE
The "row" class is interfereing with the "row" class in a custom stylesheet

### DISCUSSION THREAD

**Discussion by alikabeer32** [GrapesJS/grapesjs#5360] [Q&A]
I am trying to add a bootstrap css file to GrapesJS but the "row" class is interfering as it's already a class defined for the default blocks. Is there a way I can get around it? How can I disable those blocks? Or perhaps I can change the class name? Thank you.

**Comment by artf** [comment]
You can [update](https://grapesjs.com/docs/modules/Blocks.html#programmatic-usage) the `content` of the block

**Reply by alikabeer32** [reply]
I removed all those blocks but I still can't use the "row" class. It goes away when I click on something else. The class is still set, just not visible in the selector section on sidebar. Is there a way I can remove the predefined "row" class? How do I find it in the SelectorManager?

By the way it's not a priority so don't worry about it.
